WMF Framework/CollectionViewEditController.swift

```diff
@@ -408,18 +417,17 @@ public class CollectionViewEditController: NSObject, UIGestureRecognizerDelegate
         switch newValue {
         case .editing:
             areSwipeActionsDisabled = true
-            newBarButtonSystemItem.left = .cancel
+            leftBarButtonSystemItem = .cancel
             fallthrough
         case .swiping:
-            newBarButtonSystemItem.right = .done
+            rightBarButtonSystemItem = .done
         case .open:
-            newBarButtonSystemItem.right = UIBarButtonSystemItem.cancel
+            rightBarButtonSystemItem = .cancel
             fallthrough
         case .closed:
             transformBatchEditPane(for: editingState)
         case .empty:
             isBatchEditToolbarHidden = true
-            enabled = false
```

Comment:
Dumb question: Some states set the left button, some set the right button, some set none. 

I would expect all states to set all buttons… otherwise is there a potential to get the buttons in an inconsistent state? Or maybe I am missing how the button state gets cleaned up

Reply:
👍 good call, fixed